components/Buttons/src/MDCButton.m

```diff
@@ -215,6 +215,12 @@ - (void)encodeWithCoder:(NSCoder *)aCoder {
   [aCoder encodeObject:_accessibilityLabelForState forKey:MDCButtonAccessibilityLabelsKey];
 }
 
++ (void)initialize {
+  // Default background colors.
+  [[MDCButton appearance] setBackgroundColor:MDCColorFromRGB(MDCButtonDefaultBackgroundColor)
```

Comment:
please review the revert: https://github.com/material-components/material-components-ios/pull/1490